[Doc] Specify output parameters for FractionalMaxPool2d and FractionalMaxPool3d

isdanni opened the pull request:
Summary: Specify one of the output parameters must be set for FractionalMaxPool2d and FractionalMaxPool3d.

Fix: #104861

Test Plan: Please see GitHub Actions

Differential Revision: D47357240



cc @albanD @mruberry @jbschlosser @walterddr @mikaylagawarecki

mikaylagawarecki commented on torch/nn/modules/pooling.py:
```diff
@@ -741,6 +741,9 @@ class FractionalMaxPool2d(Module):
     step size determined by the target output size.
     The number of output features is equal to the number of input planes.
 
+    .. note:: Either of the `output_size` and `output_ratio` parameters must be set.
+              They cannot be set or empty at the same time.
```
```suggestion
    .. note:: Exactly one of `output_size` or `output_ratio` must be defined.
```

mikaylagawarecki commented on torch/nn/modules/pooling.py:
```diff
@@ -741,6 +741,8 @@ class FractionalMaxPool2d(Module):
     step size determined by the target output size.
     The number of output features is equal to the number of input planes.
 
+    .. note:: Exactly one of `output_size` or `output_ratio` must be defined.
```
Sorry one last fix for rendering

```suggestion
    .. note:: Exactly one of ``output_size`` or ``output_ratio`` must be defined.
```

isdanni committed:
[Doc] Specify output parameters for FractionalMaxPool2d and FractionalMaxPool3d (#104941)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/104941

Specify one of the output parameters must be set for FractionalMaxPool2d and FractionalMaxPool3d.

Test Plan: Please see GitHub Actions

Differential Revision: D47357240

fbshipit-source-id: e3ff0a96e461c4bc955fe7fc4669ae5601868e53

mikaylagawarecki reviewed (approved):
Thanks!

mikaylagawarecki commented:
@pytorchbot merge
